build/squire-raw.js

```diff
@@ -1131,6 +1131,10 @@ var rangeDoesStartAtBlockBoundary = function ( range, root ) {
         contentWalker.currentNode = startContainer;
     } else {
         contentWalker.currentNode = getNodeAfter( startContainer, startOffset );
+
+        if( !contentWalker.currentNode ) {
```

Comment:
Formatting error: there should be a space after `if`.

Please fix this and I'll merge.